app/src/main/java/com/github/studydistractor/sdp/distraction/Distraction.kt

```diff
@@ -13,14 +13,19 @@ data class Distraction(
     val name: String? = null,
     val description: String? = null,
     val lat: Double? = null,
-    val long: Double? = null) :
+    val long: Double? = null,
+    val length : Length? = null,

```

Comment:
I don't know about this one. We should dicuss it the next time we meet I think, because it is not clear on what we need to do. And at the same time, we have to merge to get points from the TAs. Maybe we merge both ideas and fix it for next sprint ?

Reply:
Yes. I seems good like that.